doc/whatsnew/2.10.rst

```diff
@@ -12,7 +12,9 @@ Summary -- Release highlights
 New checkers
 ============
 
+* Added ``unspecified-encoding``: Emitted when open() is called without specifying an encoding
 
+  Closes #3826
```

Comment:
```suggestion
  Closes #3826

```
Just a small suggestion: Can you add an empty line her? Usually there should be two empty ones before a new section.